app/exports/ministerial_report_applications_export.yml

```diff
@@ -0,0 +1,29 @@
+custom_columns:
+  subject:
+    type: string
+    description: The course subject
+    example: Criminology
+
+  applications:
+    type: integer
+    description: The number of applications for this subject
```

Comment:
Probably not for this PR, but it would be great to somehow standardise the language here and elsewhere in the data directory so that it's clear whether these are application forms or application choices